packages/plugin-conversation/src/share-activity.js

```diff
@@ -78,6 +78,20 @@ const ShareActivity = SparkPlugin.extend({
    * @returns {EventEmittingPromise}
    */
   add(file) {
+    let actions;
+
+    // ArrayBuffer might be located in an object with the actions.
+    if (has(file, `file`) && has(file, `actions`)) {
```

Comment:
ok, i guess my point is, rather than have two different behaviors, why not just make blobs identical to arraybuffers?